selenium/che-selenium-test/src/test/resources/suites/OrganizationSuite.xml

```diff
@@ -0,0 +1,34 @@
+<!--
```

Comment:
Why do we need a new suite?

Reply:
This suite need for starting tests only from organization package.

Reply:
You can try to run all tests from package with next command where two asterisks are used, as @tolusha suggested earlier: 
```
che-selenium-test/selenium-tests.sh --test=org.eclipse.che.selenium.dashboard.organization.**
```

Reply:
Remove suite if the command above works